orm/sequence.go

```diff
@@ -35,27 +35,37 @@ func (s *Sequence) NextInt(db weave.KVStore) (int64, error) {
 	return val, err
 }
 
+// Latest returns the recently returned value of the sequence. This method does
+// not modify the sequence state. Use NextVal or NextInt to acquire a sequence
+// value that was not given to anyone else.
+func (s *Sequence) Latest(db weave.KVStore) (int64, []byte, error) {
+	return s.increment(db, 0)
+}
+
 func (s *Sequence) increment(db weave.KVStore, inc int64) (int64, []byte, error) {
 	raw, err := db.Get(s.id)
 	if err != nil {
 		return 0, nil, err
 	}
-	val := decodeSequence(raw)
+	val := DecodeSequence(raw)
+	if inc == 0 {
+		return val, raw, nil
+	}
 	val += inc
-	raw = encodeSequence(val)
+	raw = EncodeSequence(val)
 	err = db.Set(s.id, raw)
 	return val, raw, err
 }
 
-func decodeSequence(bz []byte) int64 {
+func DecodeSequence(bz []byte) int64 {
```

Comment:
Any reason for this method and the method `EncodeSequence` below to be public?

Reply:
I need [`EncodeSequence`](https://github.com/iov-one/weave/pull/891/files#diff-ae08b07d3973d76aa1d2d3142a8567abR86) in one of the handlers. Although I only need `EncodeSequence` I have exposed both functions to make the change symmetric.
Should I make `DecodeSequence` private again?

Reply:
Symmetric is fine by me, although I suppose we should only expose method that are actually used somewhere.

Reply:
I can see that this is now private again, going to resolve this.